### PR TITLE
Feature/ddd 24 59

### PR DESCRIPTION
**Pull request by Zygis0321**
Added POST request to submit order and attributes

### CODE REVIEW

File: ui/src/components/Form.jsx [hunk 1]
```diff
@@ -39,6 +42,18 @@ function Form() {
...
+  function toggleAttribute(attribute) {
+    const attributeIndex = selectedAttributes.findIndex(attr => attr.id === attribute.id);
+    if (attributeIndex !== -1) {
+      setSelectedAttributes((attributes) =>
+        attributes.filter((item, index) => index !== attributeIndex)
+      );
+    } else {
+      setSelectedAttributes((attributes) => [...attributes, attribute]);
+    }
+    console.log(selectedAttributes);
```
Comment (giplonis) [inline ui/src/components/Form.jsx]: _nitpick_: not important right now, but keep in mind to remove stuff like this in the future projects 😄

**Review by giplonis — COMMENTED** [review]
Looks good to me. I leave the rest for @Accir

File: ui/src/components/Form.jsx (hunk 1 above)
Comment (Zygis0321) [reply ui/src/components/Form.jsx]: Oops. Good catch.

**Comment by Accir** [comment]
Looks good

**Pull request merged**